core/src/test/scala/com/salesforce/op/ModelInsightsTest.scala

```diff
@@ -474,4 +477,22 @@ class ModelInsightsTest extends FlatSpec with PassengerSparkFixtureTest {
     f0InDer3.variance shouldBe Some(3.3)
   }
 
+  it should "include raw feature distribution information calculated in RawFeatureFilter when it's included" in {
+    val wfRawFeatureDistributions = modelWithRFF.getRawFeatureDistributions()
+    val wfDistributionsGrouped = wfRawFeatureDistributions.groupBy(_.name)
+
+    // Currently, raw features that aren't explicitly blacklisted, but are not used because they are inputs to
+    // explicitly blacklisted features (eg. weight is explicitly blacklisted here, which means that height will
+    // not be added as a raw feature even though it's not explicitly blacklisted itself.
+    val insights = modelWithRFF.modelInsights(predWithMaps)
+    insights.features.foreach(f =>
+      f.distributions should contain theSameElementsAs wfDistributionsGrouped.getOrElse(f.featureName, Array.empty)
+    )
+  }
+
+  it should "not include raw feature distribution information if RawFeatureFilter is not used" in {
```

Comment:
perhaps `not include raw feature distribution info if RawFeatureFilter is not used`

Reply:
I'm missing what's different here... you want "info" instead of "information"?

Reply:
yeah ;) the point is - try to make the test name a bit shorter.